docs/developer-guide/nodejs-api.md

```diff
@@ -95,7 +95,7 @@ const { ESLint } = require("eslint");
 });
 ```
 
-### ◆ new ESLint(options)
+### new ESLint(options)
```

Comment:
I'm not sure if we want to remove `◆` 

(though, removing these will probably lead to less possible problems later on the website)

Thoughts from the team?